src/fiatconnect/slice.ts

```diff
@@ -47,11 +47,20 @@ export interface CachedQuoteParams {
   fiatType: FiatType
 }
 
+export interface TransferDetails {
+  txHash: string
+  transferId: string
+  providerId: string
+  fiatAccountId: string
+  quote: FiatConnectQuoteSuccess
```

Comment:
seems like we don't actually need a lot of the stuff from FiatConnectQuoteSuccess, like what fiat account schemas were allowed, or what kyc was required. What about just saving the stuff from the "quote" field of the [quote response](https://github.com/fiatconnect/specification/blob/main/fiatconnect-api.md#34112-responses)?

Reply:
Good call, switched to use that